apps/backend/src/routes/moments.ts

```
@@ -0,0 +1,199 @@
+  // Check if moment exists and belongs to user
+  const existing = await db
+    .select()
+    .from(moments)
+    .where(and(eq(moments.id, id), eq(moments.userId, user.id)))
+    .limit(1);
+
+  if (!existing[0]) {
+    return c.json({ error: 'Moment not found' }, 404);
+  }
+
+  const updates: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)